packages/mobile/src/store/offline-downloads/slice.ts

```diff
@@ -82,8 +82,20 @@ const slice = createSlice({
       delete state.tracks[trackId]
       delete state.downloadStatus[trackId]
     },
+    unloadTracks: (state, { payload: trackIds }: PayloadAction<string[]>) => {
+      trackIds.forEach((trackId) => {
+        delete state.tracks[trackId]
+        delete state.downloadStatus[trackId]
+      })
+    },
     doneLoadingFromDisk: (state) => {
       state.isDoneLoadingFromDisk = true
+    },
+    clearOfflineDownloads: (state) => {
+      state.collections = initialState.collections
+      state.tracks = initialState.tracks
+      state.downloadStatus = initialState.downloadStatus
+      state.isDoneLoadingFromDisk = initialState.isDoneLoadingFromDisk
```

Comment:
does `state = {...initialState}` work?
I suppose this is more explicit in case we add more unrelated state to this slice

Reply:
it really weirdly doesnt... i was really surprised

Reply:
huh. I mean this is fine too

Reply:
lets circle back on this in final qa cause i really didnt understand why it didnt work